requirements/base-requirements.in

```diff
@@ -1,7 +1,9 @@
 alembic
 architect
 attrs>=21.4
-boto3
+# Updating to a higher version breaks tests running in minio more on https://github.com/dimagi/commcare-hq/pull/32820
+# Can be updated if the tests are resolved
+boto3==1.24.96
```

Comment:
I'm not sure how aggressively pip-tools or dependabot will attempt to upgrade this library if it is not pinned? If they don't attempt to upgrade it frequently we could drop commit https://github.com/dimagi/commcare-hq/pull/32820/commits/8de1ee44ca1443b76ca520b5d991db8396044f40 and add this comment in the commit message of https://github.com/dimagi/commcare-hq/pull/32820/commits/b91c247a46def6afc7a1f7263290515460e718cb. That way upgrades are not strictly constrained, and this library may naturally upgrade to a newer version where the issue is fixed without us having to explicitly remove the pin.

Reply:
Your suggestion made so much sense in terms of not pinning boto3. I have updated the PR with the changes.